Limelight/ViewControllers/StreamFrameViewController.m

```diff
@@ -297,7 +302,7 @@ - (void)updateOverlayText:(NSString*)text {
                                            _overlayView.frame.size.height)];
         [_overlayView setText:text];
         [_overlayView sizeToFit];
-        [_overlayView setCenter:CGPointMake(self.view.frame.size.width / 2, _overlayView.frame.size.height / 2)];
+        [_overlayView setCenter:CGPointMake(self.view.frame.size.width / 2, (12 + (_overlayView.frame.size.height / 2)))];
```

Comment:
I'm guessing having a random `12` in here is not the best idea, is there a better way I could do this?